src/hooks/useDrag.tsx

```diff
@@ -184,6 +184,31 @@ export const useDrag: OnDragEndResponseWrapper = (
   return updateHomepageState(result, homepageState)
 }
 
+type UpdateHomepageType =
+  | "editor"
+  | "dropdownelem"
+  | "highlight"
+  | "announcement"
+  | `textcardcard-${number}`
+
+const isUpdateHomepageType = (value: any): value is UpdateHomepageType => {
+  if (typeof value === "string" && value.startsWith("textcardcard-")) {
+    const valArr = value.split("-")
+    const number = valArr[1]
```

Comment:
nit: it might be confusing to see `number = valArr[1]` because the variable name `number` isn't describing its usage but rather its value. 

could we use `possibleCardIndex` instead?

Reply:
Makes sense, https://github.com/isomerpages/isomercms-frontend/pull/1527/commits/e1e69692b2e410b3ef8953faf7a8b2631a30abda